code/Request.php

```diff
@@ -31,12 +31,14 @@ public function setData(array $data) {
 	 */
 	public function get() {
 		$response = [];
-		$auth     = array(
+		$useragent = 'WordPress/' . get_bloginfo( 'version' ) . '; ' . get_bloginfo( 'url' ) . '; smaily-for-wp/' . SMLY4WP_PLUGIN_VERSION;
```

Comment:
Formatting: equals signs not aligned.

Reply:
Still missing correct alignment from line 33 (`$response` variable declaration) and 41 (`wp_remote_get` call). If you don't want to align them as it may make readability worse, then just add an empty line before/after the variable.